templates/vue/cypress/support/commands.js

```diff
@@ -166,6 +166,14 @@ Cypress.Commands.add("submitModal", () => {
   cy.getByTestId("node-modal", { timeout: 10000 }).should("not.be.visible")
 })
 
+Cypress.Commands.add("submitModalWithError", () => {
+  cy.server()
+  cy.route("PUT", `**/nodes/**/permissions`).as("editPermissions")
+
+  cy.getByTestId("submit-node-modal").click()
+  cy.getByTestId("node-modal", { timeout: 10000 }).should("be.visible")
+})
+
```

Comment:
I believe this command would still run regardless if the modal has an error or not, because the node modal is visible until the action completes (so the assertion on line 174 will immediately pass). If you want to submit the modal without waiting, I believe just calling `cy.getByTestId("submit-node-modal").click()` should suffice.

Reply:
@narendrasss yeah that's probably better, as the only reason I created the new command was to prevent waiting